scripts/ci/build-installers.sh

```diff
@@ -44,6 +44,10 @@ if [ -z "$ARGV_OPERATING_SYSTEM" ] || [ -z "$ARGV_ARCHITECTURE" ]; then
   usage
 fi
 
+if [ "${GH_TOKEN:-false}" != "false" ]; then
```

Comment:
Haha, yeah, let's go for it. After all, we don't want everyone to see that such variable is set to `1` :P